spec/factories/inventories.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :inventory do
+    
```

Comment:
Please delete or add some basic fields to factory. Do not leave it empty.

Reply:
Fixed